### ISSUE TITLE
larryseyer edits broke Live Keys Scripts

### ISSUE THREAD

**Issue by larryseyer** [ReaTeam/ReaScripts#761]
Christian,

Whatever changes you made seems to have broken Live Keys Scripts.

Also, I didn't see the Live_Inst_Misc.lua script file in the package... this has all of the functions and is necessary for everything to work.

Can we go back to my original code?

Thank you.

Larry

**Comment by cfillion** [comment]
Ah right. I messed up the include path for the `dofile` (the extra `../` that shouldn't be there). This should fix it:

```lua
local script_name = ({reaper.get_action_context()})[2]:match("([^/\\_]+)%.lua$")
local this_live_track = tonumber(script_name:match("Track (%d+)")) or -1
local path = debug.getinfo(1, 'S').source:match('^@(.+)[\\//]')

dofile(string.format('%s/larryseyer_Live keys scripts.lua', path))
Live_Inst_Main_Logic(this_live_track)
```

If that works for you I'll push an update as soon as possible (sorry about that).

> Can we go back to my original code?

The changes I made were related to having adjusted the packaging metadata and renamed/deleted files: detect `this_live_track` from the action filename + (wrongly) adjust `dofile` to match the new filenames. I didn't touch your main code in "Live keys scripts.lua".

The original PR had files named "New file 1.lua" to "New file 18.lua" and none added to the action list... so at minimum that must be fixed.

**Comment by larryseyer** [comment]
OK.  I'm not sure how to test the code before you send it out...

Is there a way to do that?   I'm very new to GitHub... so sorry.

Larry

**Comment by cfillion** [comment]
Easiest way is to locally replace the contents of all "Live keys scripts" actions in the Action List (Reset, Track 1-16) with the above code.

**Comment by cfillion** [comment]
I published the include path fix from above as v1.0.0-1 (see full diff at https://github.com/ReaTeam/ReaScripts/pull/765/files). Let me know if there are further problems with the release.
